server/apps/microservices/src/processors.ts

```diff
@@ -0,0 +1,87 @@
+import {
```

Comment:
what is the different of all the processors put in here vs ion `processor/video-transcode.processor.ts`?

Reply:
These ones i have moved to the domain, so all they have is a single line which calls a service in the domain. 

The video transcode one will eventually do the same thing.

I thought about only having one file with all the processors since they are so trivial now, but we can still use one per file is your prefer. All that they do now is wire the processor decorator to the single domain call, so in my mind it is basically a mapping file of job names to service calls.